pkg/storaged/filesystem/mounting-dialog.jsx

```diff
@@ -42,9 +42,9 @@ import {
 
 const _ = cockpit.gettext;
 
-export function mounting_dialog(client, block, mode, forced_options) {
+export function mounting_dialog(client, block, mode, forced_options, subvol) {
```

Comment:
Warning against duplicate mount points doesn't seem to work for btrfs. Not sure why.

Reply:
Ah, that was because I have a duplicate entry that is not associated with any existing block device anymore. My fault. (Cockpit _could_ help, but this is independent from btrfs.)

Reply:
However, there are spurious duplicate mount point rejections with multiple btrfs devices.

For example, if I have a btrfs volume on both sda and sdb, and try to mount it, Cockpit will refuse that because "the mount point is already used for sdb". It doesn't consider that sda and adb are actually the same filesystem, and not two separate ones.